governator-core/src/test/java/com/netflix/governator/ProvisionMetricsModuleTest.java

```diff
@@ -30,13 +48,12 @@ protected void configure() {
                     protected void configure() {
                     }
                 })
-            .traceEachElement(new ProvisionListenerTracingVisitor())
             .createInjector()) {
             
             ProvisionMetrics metrics = injector.getInstance(ProvisionMetrics.class);
             LoggingProvisionMetricsVisitor visitor = new LoggingProvisionMetricsVisitor();
             metrics.accept(visitor);
-            Assert.assertTrue(visitor.getElementCount() == 0);
+            Assert.assertTrue(visitor.getElementCount() != 0);
```

Comment:
The default behavior changed.  Previously it was a no-op.  Now the default is to collect metrics.  So the element count would be non-zero because Guice provisioned something and we tracked the metrics for it.